### PR TITLE
New Crowdin translations

### PR DESCRIPTION
**Pull request by teolemon**

### CODE REVIEW

File: html/donate/hi.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="UTF-8" />
 
-    <title>Donate to Open Food Facts</title>
+    <title>ओपन फूड फैक्ट्स को दान करें</title>
```
Comment (stephanegigandet) [inline html/donate/hi.html]: I don't think it makes sense to translate "Open Food Facts". (same for other strings in Hindi below)

Comment (teolemon) [inline html/donate/hi.html]:
```suggestion
    <title>Open Food Facts को दान करें</title>
```

File: po/common/ro.po [hunk 2]
```diff
@@ -748,7 +750,7 @@ msgstr "Condiții de utilizare"
 # Do not translate without having the same exact string in the Tags template. Do not use spaces, special characters, only alphanumeric characters separated by hyphens
 msgctxt "footer_terms_link"
 msgid "/terms-of-use"
-msgstr ""
+msgstr "/termeni-si-conditii"
```
Comment (teolemon) [inline po/common/ro.po]:
```suggestion
msgstr ""
```

File: po/common/ro.po [hunk 3]
```diff
@@ -735,7 +737,7 @@ msgstr "Presa"
 # Do not translate without having the same exact string in the Tags template. Do not use spaces, special characters, only alphanumeric characters separated by hyphens
 msgctxt "footer_press_link"
 msgid "/press"
-msgstr ""
+msgstr "/presa"
```
Comment (teolemon) [inline po/common/ro.po]:
```suggestion
msgstr ""
```

File: po/common/ro.po [hunk 4]
```diff
@@ -757,11 +759,11 @@ msgstr "Cine suntem noi"
 # Do not translate without having the same exact string in the Tags template. Do not use spaces, special characters, only alphanumeric characters separated by hyphens
 msgctxt "footer_who_we_are_link"
 msgid "/who-we-are"
-msgstr ""
+msgstr "/cine-suntem-noi"
```
Comment (teolemon) [inline po/common/ro.po]:
```suggestion
msgstr ""
```

**Pull request merged**